mapping/PANCAN-mutation/map-mutations.ipynb

```diff
@@ -0,0 +1,515 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Convert mutation gene symbol labels to Entrez IDs  \n",
+    "Goal: Relabel the mutation data frame with Entrez IDs instead of gene names, by mapping a combination of chromosome and gene symbol to Entrez ID. The NCBI file downloaded and read in the next cell contains the Entrez ID - gene symbol pairs we will use to do so."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import pandas\n",
+    "from urllib.request import urlretrieve\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Creating Mappings"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>entrez_id</th>\n",
+       "      <th>chr</th>\n",
+       "      <th>symbol</th>\n",
+       "      <th>Synonyms</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>1</td>\n",
+       "      <td>19</td>\n",
+       "      <td>A1BG</td>\n",
+       "      <td>A1B|ABG|GAB|HYST2477</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>2</td>\n",
+       "      <td>12</td>\n",
+       "      <td>A2M</td>\n",
+       "      <td>A2MD|CPAMD5|FWP007|S863-7</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   entrez_id chr symbol                   Synonyms\n",
+       "0          1  19   A1BG       A1B|ABG|GAB|HYST2477\n",
+       "1          2  12    A2M  A2MD|CPAMD5|FWP007|S863-7"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "base_url = 'ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/'\n",
+    "map_name = 'Homo_sapiens.gene_info.gz'\n",
+    "map_url = base_url + map_name\n",
+    "path = os.path.join('download', map_name)\n",
```

Comment:
Can you use `os` to check if the file exists? if it doesn't, you should download. If not, file should not be downloaded